upup/models/cloudup/resources/addons/nodelocaldns.addons.k8s.io/k8s-1.12.yaml.template

```diff
@@ -192,3 +192,24 @@ spec:
           items:
             - key: Corefile
               path: Corefile.base
+---
```

Comment:
We don't ship these headless services typically used by prom operator as we don't ship prom operator either.

Also, `PodMonitor` avoids the need for these services.